src/components/ConfigureEmail.tsx

```diff
@@ -107,65 +100,72 @@ export const ConfigureEmail = (props: ConfigureEmailProps) => {
-
+  const emails = currentProfile?.emails
   return (
-    <div className="bootstrap-4-backport">
+    <div>
       {emails?.map(email => {
-        if (email === primaryEmail) {
+        if (email === primaryEmail?.email) {
           return (
-            <div key={email}>
-              <strong>{email} (Primary)</strong>
-              <button onClick={() => setEditEmail(!editEmail)}>
-                <img src={EditIcon} alt="edit icon" />
-              </button>
-            </div>
+            <Box
+              key={email}
+              sx={{ padding: '15px 0px', borderBottom: '1px solid #dcdcdc' }}
+            >
+              {email}{' '}
+              <Chip
+                sx={{ color: '#fff', marginLeft: '10px' }}
+                label="PRIMARY"
+                color="warning"
+              />
+            </Box>
           )
         } else {
           return (
-            <div key={email}>
-              {email}
+            <Box
+              key={email}
+              sx={{
+                display: 'flex',
+                padding: '15px 0px',
+                borderBottom: '1px solid #dcdcdc',
+              }}
+            >
+              <Box sx={{ flexGrow: 1, marginRight: '10px' }}>{email}</Box>
               <Button
-                style={btnStyle}
-                variant="secondary"
+                variant="outlined"
                 onClick={e => changePrimaryEmail(e, email)}
               >
-                Make primary
+                Make Primary
               </Button>
-              <Button
-                style={btnStyle}
-                variant="secondary"
+              <IconButton
+                aria-label="close"
                 onClick={e => deleteEmail(e, email)}
               >
-                X
-              </Button>
-            </div>
+                <DeleteIcon />
+              </IconButton>
+            </Box>
           )
         }
       })}
-      {editEmail && (
-        <form>
-          <FormGroup>
-            <FormControl
-              onChange={e => setNewEmail(e.target.value)}
-              placeholder="New email address"
-              value={newEmail}
-            />
-          </FormGroup>
+      <StyledFormControl variant="standard" margin="normal" fullWidth>
+        <InputLabel shrink>Add an email address</InputLabel>
+        <Box sx={{ display: 'flex' }}>
+          <TextField
+            variant="filled"
+            sx={{ flexGrow: 1, marginRight: '10px' }}
+            value={newEmail}
+            onChange={e => setNewEmail(e.target.value)}
+          />
           <Button
-            style={btnStyle}
             disabled={!newEmail}
+            variant={'contained'}
+            sx={{ alignSelf: 'flex-end', height: '47px' }}
             onClick={e => {
-              addEmail(e, newEmail)
-              setEditEmail(false)
+              addEmail(e)
             }}
           >
-            + Add email
+            <AddCircleTwoToneIcon /> Add
```

Comment:
it doesn't really matter in this case and most of the others unless there are specific styling issues, but MUI button has startIcon and endIcon props for adding an icon to the button.